src/ProjectNeon/Assets/Features/Combatants/TemporalStatType.cs

```diff
@@ -2,5 +2,8 @@ public enum TemporalStatType
 {
     HP = 0,
     Shield = 1,
-    Stun = 2
+    Stun = 2,
+    Stellar = 3,
```

Comment:
These are not Stat Types.

You can create a new enum of TemporalStatuses. 

Stats are only for things that concern themselves with numerical values. 

Stun also isn't a TemporalStatType.